games-action/minetest/minetest-0.4.15.ebuild

```diff
@@ -0,0 +1,139 @@
+# Copyright 1999-2016 Gentoo Foundation
```

Comment:
please use `repoman full` to check that the ebuild has no QA violations, this should be 2017